components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift

```diff
@@ -117,27 +119,29 @@ open class MaskedTransitionTypicalUseSwiftExample: UIViewController {
 
   var transitionController: MDCMaskedTransitionController? = nil
   func didTapFab(fab: UIView) {
-    let target = targets[tableView.indexPathForSelectedRow!.row]
-    let vc = target.viewControllerType.init()
-
-    if #available(iOS 11.0, *), target.useSafeAreaInsets {
-      vc.additionalSafeAreaInsets = view.safeAreaInsets
-    }
-
-    vc.view.autoresizingMask = target.autoresizingMask
-
-    // Customize the transition
-    let transitionController = MDCMaskedTransitionController(sourceView: fab)
-    if target.calculateFrame != nil {
-      transitionController.calculateFrameOfPresentedView = target.calculateFrame
-      vc.modalPresentationStyle = .custom
+    if let indexPathForSelectedRow = tableView.indexPathForSelectedRow {
```

Comment:
tip: by changing this into a `guard let` and returning early you can remove the added block and indentation caused by the conditional.

Reply:
Oh yes, I like that idea.